core/server/api_container/server/startosis_engine/kurtosis_instruction/tasks/tasks_shared.go

```diff
@@ -58,15 +61,22 @@ const (
 	tiniEnabled = true
 )
 
+var defaultAcceptableCodes = []int64{
```

Comment:
oh looks like you already have this; should this be used up above, when you're creating the default RunPython/RunSh objects?

Reply:
yep that is what is populated during interpretation time if the user doesn't supply them